pkg/volume/csi/csi_block.go

```diff
@@ -128,13 +132,25 @@ func (m *csiBlockMapper) SetUpDevice() (string, error) {
 		}
 	}
 
-	// create globalMapPath before call to NodeStageVolume
+	// setup path globalMapPath and block file before call to NodeStageVolume
 	if err := os.MkdirAll(globalMapPath, 0750); err != nil {
 		glog.Error(log("blockMapper.SetupDevice failed to create dir %s: %v", globalMapPath, err))
 		return "", err
 	}
 	glog.V(4).Info(log("blockMapper.SetupDevice created global device map path successfully [%s]", globalMapPath))
 
+	// create block device file
+	blockFile, err := os.OpenFile(globalMapPathBlockFile, os.O_CREATE|os.O_RDWR, 0750)
```

Comment:
this could be pulled out into a util function, looks like a similar thing is in MapDevice

Reply:
@davidz627 Wrapping this in another layer of indirection (func), would not buy too much here since there are no other logic involved other than opening the file.